### PR TITLE
Update for the new syntax

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,74 +66,79 @@ You can add the content to your Ballerina service as shown below. In that code s
...
+                // Check the student exceed the pass mark value
+                if (mark >= 60) {
+                    isQualified = true;
+                } else {
+                    isQualified = false;
+                }
+            }
+            req.setJsonPayload(untaint reqPayload);
+        } else if (reqPayload is error) {
+            log:printError("Invalid request payload: Not a json", err = reqPayload);
```
Comment (arukshani) [inline README.md]: This `Not a json` doesn't seem to be correct here. WDYT? This can be a json but the parser can fail. Just because the json is invalid it doesn't seem right to say its not a json. Better to rephrase this to say either its an invalid payload or parser error occurs or something along the line

File: README.md [hunk 2]
```diff
@@ -66,74 +66,79 @@ You can add the content to your Ballerina service as shown below. In that code s
...
-    filterMarks (endpoint caller, http:Request request) {
-        http:Response response;
-
-        // Get the JSON payload from the request
-        json reqPayload = check request.getJsonPayload();
-
-        // Get the information of the subjects
-        string stdName = check <string>reqPayload.name;
-        json subjects = reqPayload.subjects;
+    resource function filterMarks (http:Caller caller, http:Request request) {
```
Comment (arukshani) [inline README.md]: Shall we remove the space between function name and bracket `filterMarks(`

**Review by pamod — APPROVED** [review]
